src/dmclock_server.h

```diff
@@ -744,6 +764,14 @@ namespace crimson {
       }
 
 
+      inline const ClientInfo get_cli_info(ClientRec& client) const {
+	if (dynamic_cli_info_f) {
```

Comment:
Since this is the only use of the variable `dynamic_cli_info_f`, and I assume you like a well-named variable here rather than `U1`, why not put this in the line before
`constexpr bool dynamic_cli_info_f = U1;`
and then use `dynamic_cli_info_f` in your conditional?